recipe/meta.yaml

```diff
@@ -99,7 +99,9 @@ requirements:
     - palp >=2.1,<3
     # Sage uses an unreleased 2.10 pre-release
     - pari >=2.9,<3
-    - path.py >=11,<12
```

Comment:
Can you keep this?

Reply:
You mean, keep this for py3k or for all?

Reply:
Keep this for all